shopapi/views.py

```diff
@@ -60,12 +57,4 @@ def form_valid(self, form):
         # here we can send a massage to user regarding his order (SMS for example)
         return super(ProductBuyPageView, self).form_valid(form)
 
-    def get_initial(self):
```

Comment:
@vasyltsar why did you remove these lines?Is it a side effect of not doing pull into master before creating new branch?The same problem